### DISCUSSION TITLE
How to find installed versions of Secrets Store CSI Driver and Azure Key Vault provider ?

### DISCUSSION THREAD

**Discussion by mchudinov** [Azure/secrets-store-csi-driver-provider-azure#421] [Q&A]
Could someone please help with how to find versions of installed 
- Secrets Store CSI Driver version: How to find it?
- Azure Key Vault provider version: How to find it?

How to know when a new versions are available I I need to upgrade?

**Comment by aramase** [comment]
If you installed using helm charts, then you can correlate the CSI driver and AKV provider version based on the helm chart version. Here is a matrix of the versions installed based on helm release: https://github.com/Azure/secrets-store-csi-driver-provider-azure/tree/master/charts/csi-secrets-store-provider-azure#helm-chart-secrets-store-csi-driver-and-key-vault-provider-versions
